python/nav/web/templates/seeddb/edit.html

```diff
@@ -52,7 +52,11 @@
               {% endfor %}
               <tr>
                 <th></th>
-                <td><input type="submit" value="Save" class="button small"></td>
+                <td><input type="submit" value="Save" class="button small">
+                    {% if copy_url and object.pk %}
+                    <a href="{{ copy_url }}" class="button small">Copy</a>
```

Comment:
I would suggest a `title` attribute on this button, to expound on what the "copy" operation actually means. One suggestion:

> Use this room as a template for creating a new room